addons/medical_menu/stringtable.xml

```diff
@@ -45,6 +51,7 @@
         </Key>
         <Key ID="STR_ACE_Medical_Menu_openAfterTreatment_Descr">
             <English>Re-open the medical menu after succesful treatment</English>
+            <German>Öffnet das Sanitätsmenü nach einer erfolgreichen Behandlung erneut.</German>
```

Comment:
Es gibt leider noch keine einheitliche Regelung in ACE, wann und ob Satzzeichen am Ende eines übersetzten Satzes sein sollten.